braze-templates/2-braze-dashboard-simple-modal/index.html

```diff
@@ -347,14 +355,30 @@ <h2 class="modal__sub-title">Sub head maybe here that is not too long</h2>
                 <p class="modal__text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
                     incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam.</p>
                 <div class="modal__button-wrap">
-                    <a href="#" onclick="appboyBridge.logClick('1');" class="modal__btn modal__btn--secondary">Secondary
+                    <a href="#" class="modal__btn modal__btn--secondary" data-button-id="1">Secondary
                         Action</a>
-                    <a href="#" onclick="appboyBridge.logClick('0');" class="modal__btn modal__btn--main">Primary
+                    <a href="#" class="modal__btn modal__btn--main" data-button-id="0">Primary
                         Action</a>
                 </div>
             </div>
         </div>
     </div>
+
+    <script>
+        document.querySelectorAll('.close-button').forEach(function (node) {
```

Comment:
I don't see anything with this class. Seems like the below listener should cover it since some of the buttons have an id of `close` anyway